mobile-widgets/qmlmanager.cpp

```diff
@@ -869,13 +869,11 @@ void QMLManager::commitChanges(QString diveId, QString date, QString location, Q
 		d->suit = strdup(qPrintable(suit));
 	}
 	if (myDive->buddy() != buddy) {
-		if (myDive->buddy().contains(",")) {
-			if (!buddy.contains(tr("Multiple Buddies"))) {
-				diveChanged = true;
-				free(d->buddy);
-				d->buddy = strdup(qPrintable(buddy));
-			}
-		} else {
+		if (buddy.contains(",") && !buddy.contains(QRegExp("(,\\s)"))){
```

Comment:
So this triggers on a list of buddies that doesn't have a space after a comma, right? I'm not quite sure why the regex has the parenthesis... can you explain this a bit more?
But for example, "Linus,Joe, John" will pass that condition